services/frontend-v3/src/i18n/en_CA.json

```
@@ -385,12 +383,14 @@
   "sign.out": "Logout",
   "home": "Home",
 
-  "profile.addFriend": "Add Friend",
-  "profile.removeFriend": "Remove Friend",
-  "profile.removeFriend.confirm": "Are you sure you want to remove this user from your friend group ?",
+  "profile.addConnection": "Add connection",
+  "profile.removeConnection": "Remove connection",
+
+  "profile.privateGroup": "Private Group",
```

Make it lowercase https://github.com/CDH-Studio/UpSkill/pull/267#issuecomment-634277631
```suggestion
  "profile.privateGroup": "Private group",
```